ci/docker/conda-integration.dockerfile

```diff
@@ -27,8 +27,10 @@ ARG go=1.15
 
 COPY ci/conda_env_archery.yml /arrow/ci/
 RUN conda install -q \
+        --file arrow/ci/conda_env_cpp.yml \
         --file arrow/ci/conda_env_archery.yml \
         numpy \
+        compilers \
```

Comment:
If it's not useful for `conda-cpp`, is it useful here?

Reply:
It should be [in the conda-cpp](https://github.com/apache/arrow/pull/9905/files#diff-77e83f3866a7b845e723c3eb031ba36dcafc14d833ace16baecaae1a2056cf9eR29) image.

Reply:
Ah oops, sorry!